ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -44,10 +44,28 @@
           <!-- Home Sections Review -->
           <template v-if="true">
             <section class="py-6" id="review-home-sections">
-              <h2 class="px-7">Home Sections</h2>
+              <h3 class="px-7">Home Sections</h3>
               <HomeSections class=" mt-n4 px-7 py-0" :isReviewMode="true" />
             </section>
           </template>
+
+          <div class="px-4">
+            <v-divider />
+          </div>
+
+          <!-- Other Information Review -->
+          <v-row class="pa-6">
+            <v-col cols="3">
+              <h3>Other Information</h3>
+            </v-col>
+            <v-col cols="9">
+              <div>
+                <p>
+                  {{ getMhrRegistrationOtherInfo || '(Not Entered)' }}
+                </p>
+              </div>
+            </v-col>
+          </v-row>
```

Comment:
Nice, i like the style for readability. 
You could probably put the `<p>` tag on one line?
Also is the `<div>` wrapper required for the p tag?